data/strings/strings.txt

```diff
@@ -24105,7 +24109,7 @@
 
   [button_layer_isolines]
     tags = android,ios
-    en = Terrain
+    en = Isolines
```

Comment:
isolines is sort of accurate technically in a non-helpful way.  I found "terrain" strange.    The normal term in en_US is "contour lines".  I think that's the term in en_GB also (but I'm a Yank).

Reply:
agreed, changed to contour lines